src/acceptance/app/dynamic_policy_test.go

```diff
@@ -148,4 +151,74 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 		})
 
 	})
+
+	Context("when scaling by responsetime", func() {
+
+		JustBeforeEach(func() {
+			bindService := cf.Cf("bind-service", appName, instanceName, "-c", policy).Wait(cfg.DefaultTimeoutDuration())
+			Expect(bindService).To(Exit(0), "failed binding service to app with a policy ")
+			doneChan = make(chan bool)
+		})
+
+		AfterEach(func() {
+			unbindService := cf.Cf("unbind-service", appName, instanceName).Wait(cfg.DefaultTimeoutDuration())
+			Expect(unbindService).To(Exit(0), "failed unbinding service from app")
+			ticker.Stop()
+			close(doneChan)
+		})
+
+		Context("when responsetime is greater than scaling out threshold", func() {
+			BeforeEach(func() {
+				policy = generateDynamicScaleOutPolicy(1, 2, "responsetime", 3000)
+				initialInstanceCount = 1
+			})
+
+			It("should scale out", func() {
+				finishTime := time.Duration(interval*2)*time.Second + 5*time.Minute
+				ticker = time.NewTicker(10 * time.Second)
+				go func(chan bool) {
+					defer GinkgoRecover()
+					for {
+						select {
+						case <-doneChan:
+							return
+						case <-ticker.C:
```

Comment:
I guess we can just remove ticker, to continuously curl the app. With the control of sleep time, we can roughly control the response time and throughput